src/bridge/connectors/connector_0.rs

```diff
@@ -18,6 +18,13 @@ pub struct Connector0 {
 
 impl Connector0 {
     pub fn new(network: Network, n_of_n_taproot_public_key: &XOnlyPublicKey) -> Self {
+        Self::new_for_validation(network, n_of_n_taproot_public_key)
+    }
+
+    pub fn new_for_validation(
```

Comment:
Hmm, I just did the opposite in PR #78 - I removed the last occurrences of `new_for_validation` on connectors. There is no difference in the logic between `new` and that, so I think we can just keep one of them and use it whenever we need to instantiate a connector. Whether it's for validation or when creating a new graph.

Reply:
right, I removed `new_for_validation` fn from connectors I had modified earlier